data/scrape/lib/generate_summary.py

```diff
@@ -1,25 +1,16 @@
-#!/usr/bin/env python
-
 import csv
 import json
 import logging
 import operator
 import os
-
 from typing import Any, Dict
 
-from data.scrape.lib.codes import (
-    LANGUAGES_PATH,
-    README_PATH,
-    LANGUAGES_SUMMARY_PATH,
-    TSV_DIRECTORY,
-)
+import pandas as pd
```

Comment:
I know this alias is something people do online but I never understood why; It's just a few saved characters. Do `import pandas` instead please.